doc/adding-a-language.md

```diff
@@ -123,6 +123,14 @@ programming language and use a constraint to select large projects, such
 as "> 100 forks". Collect the repository URLs and put them into
 `projects.txt`.
 
+Generating a Boilerplate
+--
+
+After you have pushed your ocaml-tree-sitter changes to the main branch, do the following:
+1. In `ocaml-tree-sitter/lang/Makefile`, add language under 'SUPPORTED_LANGUAGES' and 'STAT_LANGUAGES'.
+2. In `ocaml-tree-sitter/lang` directory, run `./release X`. This will automatically 
+   add boilerplate code to `ocaml-tree-sitter-lang`. 
```

Comment:
it's not boilerplate. There is a Boilerplate.ml file, but the rest, like CST.ml and the parser.c are auto-generated code for parsing, CST type definitions, converter from tree-sitter JSON to CST.ml, etc.

Reply:
I would just write "generated code" instead of "boilerplate code". They're all quite repetitive and large files, but only one of them is meant to be hand-edited (`Boilerplate.ml`).

Reply:
done!